apps/screloc/main.cpp

```diff
@@ -70,25 +67,48 @@ class Reloc : public Client::Application {
+			commandline().addOption("Input", "origin-id,O",
+			                        "Reprocess the origin and send a message.",
+			                        &_originIDs);
+			commandline().addOption("Input", "locator",
+			                        "The locator type to use.", &_locatorType, false);
+			commandline().addOption("Input", "profile",
+			                        "The locator profile to use.", &_locatorProfile, false);
+			commandline().addOption("Input", "use-weight",
+			                        "Use current picks weight.", &_useWeight, true);
+			commandline().addOption("Input", "ep",
+			                        "Event parameters XML file for offline "
+			                        "processing of all contained origins. This "
+			                        "option should not be mixed with --dump.",
+			                        &_epFile);
+			commandline().addOption("Input", "replace",
+			                        "Used in combination with --ep and defines "
+			                        "if origins are to be replaced by their "
+			                        "relocated counterparts or just added to the output.");
+			commandline().addOption("Input", "drop-failure",
+			                        "Used in combination with --replace/--ep "
+			                        "and drops from the output the origins for "
+			                        "which the relocation failed.");
 			commandline().addGroup("Output");
-			commandline().addOption("Output", "origin-id-suffix", "create origin ID from that of the input origin plus the specfied suffix", &_originIDSuffix);
-			commandline().addOption("Output", "evaluation-mode", "evaluation mode of the new origin (AUTOMATIC or MANUAL)", &_originEvaluationMode, true);
+			commandline().addOption("Output", "origin-id-suffix",
+			                        "Create origin ID from that of the input "
+			                        "origin plus the specfied suffix.",
+			                        &_originIDSuffix);
+			commandline().addOption("Output", "evaluation-mode",
+			                        "Evaluation mode of the new origin (AUTOMATIC or MANUAL).",
+			                        &_originEvaluationMode, true);
 			commandline().addGroup("Profiling");
-			commandline().addOption("Profiling", "measure-relocation-time", "measure and log the time it takes to run each relocation");
-			commandline().addOption("Profiling", "repeated-relocations", "improve measurement of relocation time by running each relocation multiple times", &_repeatedRelocationCount);
+			commandline().addOption("Profiling", "measure-relocation-time",
+			                        "Measure and log the time it takes to run each relocation.");
+			commandline().addOption("Profiling", "repeated-relocations",
+			                        "Improve measurement of relocation time by "
+			                        "running each relocation multiple times.",
+			                        &_repeatedRelocationCount);
```

Comment:
Large code reformatting that has nothing to do with the current PR should go into a different commit. I don't mind to keep the commit in the same PR, but putting them in a different commit would greatly help the code maintenance, back-porting and review.

Reply:
will do so

Reply:
created separate commits